extensions/interactions/EndExploration/EndExploration.js

```diff
@@ -57,7 +57,7 @@ oppia.directive('oppiaInteractiveEndExploration', [function() {
           // Display a message if any author-recommended explorations are
           // invalid.
           var explorationId = explorationContextService.getExplorationId();
-          $http.get('/explorationsummarieshandler/data', {
+          $http.get(EXPLORATION_SUMMARY_DATA_URL_TEMPLATE, {
```

Comment:
Will correct, need to test. In The Meantime, I'll post all of the fixes for your review.